docs/content/what-is-a-child-workflow-execution.md

```diff
@@ -9,39 +9,60 @@ tags:
-- A Parent Workflow Execution can also propagate Cancellation Requests and Terminations to Child Workflow Executions.
-  This is determined by a [Parent Close Policy](/docs/content/what-is-a-parent-close-policy).
-- A Workflow Execution can be both a Parent and a Child Workflow Execution.
-- If a Child Workflow Execution uses Continue As New, from the Parent Workflow Execution's perspective it is still a single Child Workflow Execution.
-- If a Parent Workflow Execution uses Continue As New without awaiting on the result of the Child Workflow Execution, then the Child Workflow Execution becomes an orphan.
-  The Child Workflow Execution must have an Abandon Parent Close Policy when then happens or else it will Terminate when the Parent Continues As New.
-  An orphan Child Workflow Execution will not have any Cancellation Requests or Terminations propagated to it.
+A Workflow Execution can be both a Parent and a Child Workflow Execution, as any Workflow can spawn another Workflow.
 
 <CenteredImage
 imagePath="/diagrams/parent-child-workflow-execution-relationship.svg"
 imageSize="100"
 title="Parent & Child Workflow Execution entity relationship"
+/>
+
+A Parent Workflow Execution must await on the Child Workflow Execution to spawn.
+The Parent can optionally await on the result of the Child Workflow Execution.
+Consider the Child's [Parent Close Policy](/docs/content/what-is-a-parent-close-policy) if the Parent does not await on the result of the Child, which includes any use of Continue As New by the Parent.
+
+When a Parent Workflow Execution reaches a Closed status, the Server will propagate Cancellation Requests or Terminations to Child Workflow Executions depending on the Child's [Parent Close Policy](/docs/content/what-is-a-parent-close-policy).
+
+<CenteredImage
+imagePath="/diagrams/parent-close-policy.svg"
+imageSize="75"
+title="Parent Close Policy entity relationship"
+/>
+
+If a Child Workflow Execution uses Continue As New, from the Parent Workflow Execution's perspective the entire chain of Runs is treated as a single execution.
+
+<CenteredImage
+imagePath="/diagrams/parent-child-workflow-execution-with-continue-as-new.svg"
+imageSize="100"
+title="Parent & Child Workflow Execution entity relationship with Continue As New"
 legend={[
 ["*","Last Workflow Execution in the chain"]  
 ]}
 />
 
-**Consider Workflow Execution Event History size limits**: An individual Workflow Execution has an Event History size limit, which imposes a couple of considerations for using Child Workflows.
+### When to use Child Workflows
+
+**Consider Workflow Execution Event History size limits.**
+
+An individual Workflow Execution has an Event History size limit, which imposes a couple of considerations for using Child Workflows.
 On one hand, because a Child Workflow Executions have their own Event Histories, they are often used to partition large workloads into smaller chunks.
```

Comment:
```suggestion

On one hand, because a Child Workflow Executions have their own Event Histories, they are often used to partition large workloads into smaller chunks.
```